src/Backend/Modules/Profiles/Actions/Index.php

```diff
@@ -130,14 +130,26 @@ private function loadDataGrid(): void
         );
         $this->dgProfiles->setColumnFunction('htmlspecialchars', ['[display_name]'], 'display_name', false);
 
+        // check if
+        $groups = BackendProfilesModel::getGroups();
```

Comment:
Would't I be more logical to have the $actions populated at the start. So:

```
$actions = [
  'delete' => BL::getLabel('Delete'),
];

if (count($groups) > 0) {
  $actions['addToGroup'] = BL::getLabel('AddToGroup');
}
```

Reply:
Updated in 22b604e